pandas/tools/tile.py

```diff
@@ -81,6 +83,17 @@ def cut(x, bins, right=True, labels=None, retbins=False, precision=3,
     array([1, 1, 1, 1, 1], dtype=int64)
     """
     # NOTE: this binning code is changed a bit from histogram for var(x) == 0
+    # for handling the cut for datetime and timedelta objects
+
+    dtype = None
+    if is_timedelta64_dtype(x):
+        x = x.astype(np.int64)
+        dtype = np.timedelta64
+
+    if is_datetime64_dtype(x):
```

Comment:
and will that done be in the same same manner as well? (by converting to int64?)